js/lib/explore.min.js

```diff
@@ -117,8 +117,8 @@
 
 	/*!
 	 * Stickyfill -- `position: sticky` polyfill
-	 * v. 1.1.10 | https://github.com/wilddeer/stickyfill
-	 * Copyright Oleg Korsunsky | http://wd.dizaina.net/
+	 * v. 1.1.11 | https://github.com/18F/stickyfill
+	 * Copyright Brian Hedberg | 
```

Comment:
As far as I know, 18F frowns on this because I made these updates on 18F time. I can investigate how to get around this if you think it is necessary.